homeassistant/components/control4/strings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "The Control4 controller associated with this account has already been configured."
+    },
+    "error": {
+      "cannot_connect": "Couldn't connect to the local controller. Check your hostname and login details.",
+      "invalid_auth": "Invalid Control4 username or password.",
+      "unknown": "An unknown error occured. Please report log info."
+    },
+    "step": {
+      "user": {
+        "data": {
+          "host": "Controller hostname",
+          "password": "Password",
+          "username": "Username/Email"
+        },
+        "title": "Control4",
+        "description": "Please enter your Control4 account details and the hostname of your local controller."
+      }
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "light_transition_time": "Default light brightness transition time (seconds)",
+          "light_cold_start_transition_time": "Default light cold start time (seconds)",
+          "scan_interval": "Seconds between updates"
```

Comment:
Why would we make this configurable ? We should pick the most aggressive option that the hub can handle. We shouldn't leave it up to the user to pick (and mess up)

Reply:
I put this in because Control4 systems can vary in size greatly (from tens to hundreds to even thousands?) of devices, so a polling rate that works for most may lag a older controller with many devices attached to it.

Reply:
Could we make some logic where we say

```
if devices < 50:
  return 5 seconds
elif devices < 200:
  return 15 seconds
else:
  return 1 minute
```

?

Reply:
I would rather not, mainly because I'm not sure how big a Control4 system would have to be before slowing down. It probably also depends on what specific features are being used (for example, surveillance video recording or movie playback).

I've changed the default to be more aggressive (5 seconds),  which I think should be appropriate for most systems. Most users should never have to change the scan_interval option now.